src/extensions/mod.rs

```diff
@@ -56,7 +59,7 @@ pub trait Extension: Sync + Send + 'static {
     fn validation_end(&mut self) {}
 
     /// Called at the begin of the execution.
-    fn execution_start(&mut self) {}
+    fn execution_start<'a>(&mut self) {}
```

Comment:
It looks quite strange here. 🙂

Reply:
Oups it was from a previous implementation, need to delete